src/controllers/books.controller.js

```diff
@@ -0,0 +1,103 @@
+const { isObjEmpty } = require('../utils');
+const model = require('../models/books.model');
+
+const getAllBooks = async (req, res) => {
+  try {
+    const books = await model.getAllBooks(req);
+
+    res.status(200).json({
+      books,
+    });
+  } catch (error) {
+    console.error('[ERROR]', error);
+    res.sendStatus(500);
+  }
+};
+
+const getBookById = async (req, res) => {
+  const bookId = req.params.id;
+  try {
+    const book = await model.getBookById(bookId);
+
+    isObjEmpty(book)
+      ? res
+          .status(404)
+          .json({ error: 'A book with the provided ID does not exist' })
+      : res.status(200).json({
+          book,
+        });
```

Comment:
Try not to overuse ternary - the general rule is if you can't fit it on one line, it should be an `if` statement. You can avoid having to use `else` by restructuring this to something like:

```js
if (isObjEmpty(book)) {
  return res.status(404).json({ error: 'A book with the provided ID does not exist' })
}

res.status(200).json({ book })
```